docs/website/docs/dlt-ecosystem/destinations/filesystem.md

```diff
@@ -699,4 +699,27 @@ You will also notice `init` files being present in the root folder and the speci
 
 **Note:** When a load generates a new state, for example when using incremental loads, a new state file appears in the `_dlt_pipeline_state` folder at the destination. To prevent data accumulation, state cleanup mechanisms automatically remove old state files, retaining only the latest 100 by default. This cleanup process can be customized or disabled using the filesystem configuration `max_state_files`, which determines the maximum number of pipeline state files to retain (default is 100). Setting this value to 0 or a negative number disables the cleanup of old states.
 
+## Troubleshooting
+When running your pipeline, you might encounter an error like `[Errno 36] File name too long Error`. This error occurs because the generated file name exceeds the maximum allowed length on your filesystem.
+
+### Solution
+To prevent this, you can set the `max_identifier_length` parameter for your destination. This truncates all identifiers (including filenames) to a specified maximum length.
```

Comment:
```suggestion
To prevent the file name length error, set the `max_identifier_length` parameter for your destination. This truncates all identifiers (including filenames) to a specified maximum length.
```
Rewrite for clarity